frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -104,7 +104,7 @@ export const Marketplace = () => {
   resources.push({
     isList: true,
     kind: referenceForModel(PackageManifestModel),
-    namespace: undefined, // shows operators from all-namespaces - when backend is hooked up we will use 'marketplace'
+    namespace: 'openshift-marketplace',
```

Comment:
It would be nice to not hardcode the namespace here: we did this with OLM initially and it causes problems with local dev/e2e testing.